js/bitstamp.js

```diff
@@ -999,13 +999,35 @@ module.exports = class bitstamp extends Exchange {
             return; // fallback to default error handler
         if ((body[0] === '{') || (body[0] === '[')) {
             // fetchDepositAddress returns {"error": "No permission found"} on apiKeys that don't have the permission required
-            let error = this.safeString (response, 'error');
-            let exceptions = this.exceptions;
-            if (error in exceptions) {
-                throw new exceptions[error] (this.id + ' ' + body);
-            }
+            response = this.parseJson (body);
```

Comment:
This is now coming from the outside in the arguments, so we don't need this line anymore:`if (response !== undefined)` – then it's a properly parsed json, otherwise we should fallback to the default handler. So, we probably need to rewrite this line and the lines above to simplify this a bit. All of the above is there for historical reasons mostly... See `kucoin.js` for the beginning of `handleErrors()`.

Reply:
I think it'll be fine with just checking `if (response === undefined) return;` and then otherwise going on to parse response then. If response is always parsed json then we can check for error and then continue.